src/syzygy/tbcore.cpp

```diff
@@ -39,15 +45,20 @@ static TBHashEntry TB_hash[1 << TBHASHBITS][HSHMAX];
 
 static DTZTableEntry DTZ_table[DTZ_ENTRIES];
 
-static uint64_t calc_key_from_pcs(int *pcs, bool mirror);
+static uint64_t calc_key_from_pcs(uint8_t* pcs, bool mirror);
```

Comment:
Yes, avoiding a bunch of wild cast

On Tue, Apr 19, 2016 at 1:31 PM, lucasart <notifications@github.com> wrote:

> In src/syzygy/tbcore.cpp
> <https://github.com/official-stockfish/Stockfish/pull/641#discussion_r60214446>
> :
>
> > @@ -39,15 +45,20 @@ static TBHashEntry TB_hash[1 << TBHASHBITS][HSHMAX];
> >
> >  static DTZTableEntry DTZ_table[DTZ_ENTRIES];
> >
> > -static uint64_t calc_key_from_pcs(int *pcs, bool mirror);
> > +static uint64_t calc_key_from_pcs(uint8_t* pcs, bool mirror);
>
> any reason to change pcs from int to uint8_t ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/official-stockfish/Stockfish/pull/641/files/decb8b383f40787aab2522f2716bca6c7fdc342d#r60214446>
>